mlcubes/data_preparation/project/sanity_check.py

```diff
@@ -0,0 +1,50 @@
+import yaml
+import argparse
+import pandas as pd
+
+from stages.utils import has_prepared_folder_structure
+
+def sanity_check(data_path: str, labels_path: str, report_df: pd.DataFrame):
```

Comment:
@aristizabal95 I see the report file is an input, but it's not used. Also, the metadata path is an input to the sanity checks, but it's not used.
let's remove these from the sanity checks task?

Reply:
Yeah, we can remove them. I added them thinking of the general case, where maybe some checks need to look into the report or metadata paths. But, since we now know those files might not exist then it's better to enforce not depending on them